data/campaigns/World_Conquest/lua/map/tools/filter_converter.lua

```diff
@@ -3,7 +3,7 @@
 ---- A helper tool, that works (mostly) indpendently  ----
 ---- of the rest to convert location filter wml to    ----
 ---- the filter syntax used in this addon             ----
----- (map:get_locations)                              ----
+---- (map:find)                              ----
```

Comment:
Can you add more spaces to keep the right border aligned?

If you want you could even merge this into the previous line, it's short enough.

Reply:
I have added more spaces to "restore" this comment box..or whichever term is used to describe this...